whatsabi/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Elysia, t } from "elysia";
+import { swagger } from '@elysiajs/swagger'
+import { whatsabi } from "@shazow/whatsabi";
+import { ethers } from "ethers";
+import { logger } from "@bogeychan/elysia-logger";
+
+const port = process.env.WHATSABI__PORT;
+
+export function initApp(port: string | number) {
+    return new Elysia()
+        .use(swagger({
+            path: '/api/v1/swagger'
+        }))
+        .use(
+            logger({
+                level: "info",
+            })
+        )
+        .get("/api/v1/abi", async ( { request, log, query }) => {
+            log.info(request, "New request");
+            return await processAbi(query.address, query.provider)
```

Comment:
maybe add some log to indicate that for `address` we found abi with N amount of entries

Reply:
or `address not found`
or `no bytecode for address`
and so on

Reply:
Btw, in case if address is not a contract, then just empty array will be returned as a result and not an error